packages/server/src/routes/members/index.ts

```diff
@@ -664,6 +665,10 @@ async function _createInvitedMember(
   const newMember = {
     username,
     full_name: fullName,
+    // TODO Remove or-check once we're sure all clients have upgraded to request email on signup.
+    // Updated client will have version number 0.0.6 for Android.
+    email_address: emailAddress || "",
```

Comment:
why "" instead of something like `undefined`? If it's not actually required at the moment it should probably just be optional.

Reply:
I can use `null`. Firebase throws an error on `undefined` fields.